consumatio/external/tmdb.py

```diff
@@ -9,47 +9,70 @@
 from consumatio.gateways.season_images_to_dict import * 
 from consumatio.gateways.episode_details_to_dict import * 
 from consumatio.gateways.episode_images_to_dict import *
+from consumatio.external.db import Database
+import json
 
 class Tmdb():
-    
+
+    def __init__(self):
+        self.db = Database()
+
     def get_movie_details(self, movie_id):
-        data = requests.get(f'https://api.themoviedb.org/3/movie/{movie_id}?api_key=c4bd02adee6e73c4d17e69b039267c90&language=en-US').json()
+        query = f'https://api.themoviedb.org/3/movie/{movie_id}?api_key=c4bd02adee6e73c4d17e69b039267c90&language=en-US'
```

Comment:
Please decompose `https://api.themoviedb.org/3/` here and in the following API requests and get the `api_key` from an env variable

Reply:
I see that there is already an issue for the API keys, closing